frontend/src/constants/QuestionConstants.ts

```diff
@@ -0,0 +1,12 @@
+const editorTooltips = {
+  QUESTION_TEXT: "This is question text",
+  TEXT: "This is text",
+  IMAGE: "This is an image",
+  SHORT_ANSWER: "Users will have to type in their answers",
+  MULTI_SELECT:
+    "Users will have to select the correct responses from a list of options",
+  MULTIPLE_CHOICE:
+    "Users will have to select the best response from a list of options",
+};
+
+export default editorTooltips;
```

Comment:
copy is subject to change here, need to get finalized copy from product & client. for now, adding these as placeholders!

also, created a ticket to combine this and `StudentAssessmentConstants` later b/c i think we are duplicating, but wanna get this pr in before tmrw's demo